docs/README.md

```diff
@@ -21,6 +21,8 @@ Perform the actions from the sub-documents in the following order:
 
 * [Keystone adoption](openstack/keystone_adoption.md)
```

Comment:
In the keystone_adoption.md , there are commands to remove old endpoints and services.  We need to add barbican here too.